crates/relayer/src/chain/ckb/utils.rs

```diff
@@ -366,6 +372,13 @@ mod tests {
         let path = TempDir::new().unwrap();
         let storage: Storage<MainnetEthSpec> = Storage::new(path).unwrap();
 
+        (chain_id, updates_part_1, updates_part_2, storage)
+    }
```

Comment:
Leave this closure, the destructor of [`tempfile::TempDir`] will run automatically in [its `Drop` implement].

Then, the files of storage could be broken.

So, the result of the unit tests will be randomized.

A very important tip for using `tempfile` is that to make sure the lifetime of [`tempfile::TempDir`] is longer than the lifetime of structs which use it.

References:
- [`tempfile::TempDir`]
- [`MockStore` in CKB](https://github.com/nervosnetwork/ckb/blob/develop/util/test-chain-utils/src/mock_store.rs#L20-L27) (the comment in this piece code is useful)


p.s. I would like to push more commits base on this PR, so you don't have to update this issue, I will do that.

 [`tempfile::TempDir`]: https://docs.rs/tempfile/latest/tempfile/struct.TempDir.html
[its `Drop` implement]: https://docs.rs/tempfile/latest/tempfile/struct.TempDir.html#impl-Drop

Reply:
ok, it's fixed with your previous PR.